src/app/patient-dashboard/common/patient-info/edit-address.component.ts

```diff
@@ -3,20 +3,22 @@ import { PatientService } from '../../services/patient.service';
 import { Patient } from '../../../models/patient.model';
 import { PersonResourceService } from '../../../openmrs-api/person-resource.service';
 import { Subscription } from 'rxjs';
+import { LocationResourceService } from 'src/app/openmrs-api/location-resource.service';
 
 @Component({
   selector: 'edit-address',
   templateUrl: './edit-address.component.html',
   styleUrls: [],
 })
 export class EditAddressComponent implements OnInit, OnDestroy {
-
   public patients: Patient = new Patient({});
   public subscription: Subscription;
+  public subscription1: Subscription;
```

Comment:
```suggestion
  public locationSubscription: Subscription;
```

Reply:
Or perhaps:

```js
public subscriptions: Subscription[]  = [];
```

And then later in `getPatient`:
```js
const getPatientSubscription = ...

subscriptions.push(getPatientSubscription);
```

and then `OnDestroy` you could map through and garbage collect:

```js
  public ngOnDestroy(): void {
    if (this.subscriptions.length) {
      this.subscriptions.map(sub => sub.unsubscribe);
    }
  }
```